crates/common_enums/src/enums.rs

```diff
@@ -404,25 +404,25 @@ pub enum AuthorizationStatus {
     Unresolved,
 }
 
-// #[derive(
-//     Clone,
-//     Debug,
-//     Eq,
-//     PartialEq,
-//     serde::Deserialize,
-//     serde::Serialize,
-//     strum::Display,
-//     strum::EnumString,
-//     ToSchema,
-//     Hash,
-// )]
-// #[router_derive::diesel_enum(storage_type = "text")]
-// #[serde(rename_all = "snake_case")]
-// #[strum(serialize_all = "snake_case")]
-// pub enum SessionUpdateStatus {
-//     Success,
-//     Failure,
-// }
+#[derive(
+    Clone,
+    Debug,
+    Eq,
+    PartialEq,
+    serde::Deserialize,
+    serde::Serialize,
+    strum::Display,
+    strum::EnumString,
+    ToSchema,
+    Hash,
+)]
+#[router_derive::diesel_enum(storage_type = "text")]
```

Comment:
we do not need to derive database traits if this enum will not be stored in the database